src/Capability.php

```
@@ -0,0 +1,114 @@
+<?php
+namespace RolesCapabilities;
+
+/**
+ * Capability Class
+ */
+class Capability
+{
+    /**
+     * Capability name
+     * @var string
+     */
+    protected $_name;
+
+    /**
+     * Capability label
+     * @var string
+     */
+    protected $_label;
+
+    /**
+     * Capability description
+     * @var string
+     */
+    protected $_description;
+
+    /**
+     * Constructor method
+     * @param string $options Capability options
+     */
+    public function __construct(array $options = [])
+    {
+        $name = isset($options['name']) ? $options['name'] : '';
+        $this->setName($name);
+
+        $label = isset($options['label']) ? $options['label'] : '';
+        $this->setLabel($label);
+
+        $description = isset($options['description']) ? $options['description'] : '';
+        $this->setDescription($description);
+    }
+
+    /**
+     * toString method
+     * @return string Capability name
+     */
+    public function __toString()
+    {
+        return $this->_name;
+    }
+
+    /**
+     * Set name
+     * @param string $name Capability name
+     * @return Capability
+     */
+    public function setName($name)
+    {
+        $this->_name = $name;
```

@georgeconstantinou we should probably throw an InvalidArgumentException if the name is empty or not a string.  Since this is a public method, it's best to check here, rather than in the constructor.